code/game/machinery/rechargestation.dm

```diff
@@ -13,6 +13,10 @@
 	var/recharge_speed_nutrition
 	var/repairs
 
+/obj/machinery/recharge_station/Destroy()
+	go_out()
+	. = ..()

```

Comment:
please stick with `return ..()`

Unless you need to return the value prior to further procs, stick with explicit returning.

Reply:
Done